### PR TITLE
aruba updates

### PR DESCRIPTION
**Pull request by inphobia**
see #126

### CODE REVIEW

File: EXTRAS/reports/aruba [hunk 1]
```diff
@@ -1030,6 +1030,282 @@
...
+                  dot1dStpPortGroup2(5) type=22
+                  dot1dStpPortGroup3(6) type=22
+                  dot1dTpBridgeGroup(7) type=22
+                  dot1dTpFdbGroup(8) type=22
+                  dot1dTpGroup(9) type=22
+                  dot1dStaticGroup(10) type=22
+                  dot1dNotificationGroup(11) type=23
+                dot1dCompliances(2) type=0
+                  bridgeCompliance1493(1) type=26
+                  bridgeCompliance4188(2) type=26
```
Comment (JeroenvIS) [inline EXTRAS/reports/aruba:1308]: Strange, RFC related objects in the "reports/aruba" diff that are not in the "reports/all" ? Haven't checked in detail yet, perhaps these are copies of RFC MIBs that are relocated somewhere into the aruba tree for experimental/testing reasons?

File: EXTRAS/reports/aruba [hunk 2]
```diff
@@ -1094,7 +1370,123 @@
...
+                pethCompliances(1) type=0
+                  pethCompliance(1) type=26
+                pethGroups(2) type=0
+                  pethPsePortGroup(1) type=22
+                  pethMainPseGroup(2) type=22
+                  pethNotificationControlGroup(3) type=22
+                  pethPsePortNotificationGroup(4) type=23
+                  pethMainPowerNotificationGroup(5) type=23
+            ianaEntityMIB(216) type=24
+            uuidTCMIB(217) type=24
```
Comment (JeroenvIS) [inline EXTRAS/reports/aruba:1489]: Same comment, these are unexpected in the reports/aruba part

File: EXTRAS/reports/aruba [hunk 3]
```diff
@@ -4310,6 +4769,456 @@
...
+                              arubaWiredMstpPreviousRootBridgeID(13) type=2 hint=255a
+                              arubaWiredMstpNewRootBridgeID(14) type=2 hint=255a
+                              arubaWiredMstpRootBridgeChangeTimeStamp(15) type=2 hint=2d-1d-1d,1d:1d:1d.1d,1a1d:1d
+                              arubaWiredMstpId(16) type=16
+                        arubaWiredMstpConformance(2) type=0
+                          arubaWiredMstpGroups(1) type=0
+                            arubaWiredMstpNotificationObjectGroup(1) type=22
+                            arubaWiredMstpNotificationGroup(2) type=23
+                          arubaWiredMstpCompliances(2) type=0
+                            arubaWiredMstpNotificationCompliance(1) type=26
```
Comment (JeroenvIS) [inline EXTRAS/reports/aruba:5221]: These all look fine in the report.

**Review by JeroenvIS — COMMENTED** [review]
In general, the changes look good. And a welcome addition, because several of these MIBs are needed to create a SNMP::Info module for Aruba switches running AOS-CX software (eg 6200/6300/6400 and 8320/8325/8360/8400 series).

Only the bit about reports/aruba including large blocks of RFC related objects worries me. haven't had time to check where that comes from.

File: EXTRAS/reports/aruba (hunk 1 above)
Comment (inphobia) [reply EXTRAS/reports/aruba:1308]: good catch, i guess these might be due to new mibs importing rfc mibs?

ARUBAWIRED-CIPT-MIB:

        VlanIndex
            FROM Q-BRIDGE-MIB

**Comment by JeroenvIS** [comment]
I guess all looks good, let's merge.

**Comment by JeroenvIS** [comment]
Okay, should have payed more attention to the paths: some files have been put in the hpe/ directory and some in aruba/ - this creates an unneeded dependency between the two. It;s probably better to put all these files in aruba/

**Pull request merged**